timer/timer_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+make format
+make
+make install
+for (( del = 0; del < 30; del += 5)); do
+        sleep $del
+        iter=$(cat /sys/class/timer/inter)
+        time=$(cat /sys/class/timer/time)
+        echo -e "delay = $del\t${iter}\t${time}"
+done
+make uninstall
+
+exit $RETURN_SUCCESS
```

Comment:
What is the value of RETURN_SUCCESS?
Is this exit required?

Reply:
No, it's just code style in this example. exit $RETURN_SUCCESS directly indicates the end of script

Reply:
I suppose that using uninitialized variables is not good practice.

Reply:
I've removed this unnecessary exit call.